### ISSUE TITLE
Timeline on releasing the generative framework

### ISSUE THREAD

**Issue by chanhee-luke** [Genesis-Embodied-AI/Genesis#6]
Hi team,

I am excited to see Genesis released! I briefly played with it and it is so easy to use and fast.
I am interested in the generative framework. Do you have a timeline for when it will be available?

Thank you!

Best,
Luke Song

**Comment by twinsant** [comment]
Demo 式开源吗？

**Comment by ziyanx02** [comment]
Providing a detailed timeline is challenging due to limited contributors, but stay tuned. We will keep you informed by updating this issue as soon as we make any progress.

**Comment by wyahnke** [comment]
How can we sign up to the wait list for the generative features?

**Comment by PyroFilmsFX** [comment]
Yeah this was quite the misrepresentation to not once mention that that portion is unreleased. But also would like to be added to a waitlist.

**Comment by gauravdhiman** [comment]
Thanks for creating this issue. Is there a way to add ourself to waitlist or get notified when it gets released ?

**Comment by PyroFilmsFX** [comment]
@ziyanx02 hello?

**Comment by PyroFilmsFX** [comment]
@ziyanx02 hello?

**Comment by phazei** [comment]
Monthly bump ;)  Can't wait...

**Comment by NashvilleBrandon** [comment]
👀

**Comment by rfan-debug** [comment]
Saw your seed round. Congrats! Looking forward to releasing the generative framework!

**Comment by garylvov** [comment]
6 months and no estimate for the release of one of the key advertised use cases 🙈

Bumping +1, is there a plan or estimate of time to release the generative part as open-source?
